src/components/binding/binding-loader.js

```diff
@@ -31,16 +32,23 @@ genotet.BindingLoader.prototype.LOCUS_MARGIN_RATIO = .1;
  * @param {string} fileName Binding file name.
  * @param {string} bedName Bed name of the bed binding track.
  * @param {string} chr ID of the chromosome.
+ * @param {number} numberOfTracks Number of tracks.
  * @param {number=} opt_track Track # into which the data is loaded.
  * @override
  */
 genotet.BindingLoader.prototype.load = function(fileName, bedName, chr,
-                                                opt_track) {
+                                                numberOfTracks, opt_track) {
   var trackIndex = opt_track ? opt_track : this.data.tracks.length;
   this.data.chr = chr;
   this.loadFullTrack(trackIndex, fileName, chr);
   this.loadBed(bedName, chr, this.data.detailXMin, this.data.detailXMax);
   this.loadExons_(chr);
+
+  this.data.defaultNumberOfTracks = numberOfTracks;
```

Comment:
overviewRangeChanged denotes whether the currently loaded tracks have their entire overview range updated. Why do you think that shi will depend on the default number of tracks? I don't see any connection between them. You are now making things weird in a weird way.

- bindingRangeChanged can only be set to true when the overviewRange becomes different.
- IMO defaultNumTracks should not be even created, because nothing depend on it.